dependencies.yaml

```
@@ -442,3 +442,4 @@ dependencies:
           - pytest-cov
           - pytest-xdist
           - geopandas>=0.11.0
+          - cuspatial=23.10.*
```

```suggestion
          - cuspatial==23.10.*
```

And re-run pre-commit.